sfcli/README.md

```diff
@@ -182,81 +162,76 @@ Options:
 
 Example:
 ```sh
-sfcli --url http://localhost:8099 watchlistmember registerFromDir -w:fingera -d:"d:\Fingera Registration Photo"
+sfcli --host localhost watchlistmember registerFromDir -w:fingera -d:"d:\Fingera Registration Photo"
 ```
 
-#### softrestart
+#### camera
 
-When config values are changed, they are not reflected immediately. To load changed configurations of you need to call softrestart command.
+This command allow to create, update or view settings of a camera. Camera is domain entity which groups some data entities, workers and configs. That allows you to add live stream from IP camera and easily change processing properties.
```

Comment:
should we mention mapping between domain/config workers at all here ?

Reply:
The mapping was mentioned previously in the readme. If we expect the user of sfcli to also know database schema/interact with database directly, then this is a useful information IMO